.github/workflows/test-ui-e2e-only-desktop.yml

```diff
@@ -164,6 +191,50 @@ jobs:
         shell: bash
         run: echo "::notice title=Allure report URL::${{ steps.allure-server.outputs.report-url }}"
 
+      - name: Download Xray Reports
```

Comment:
Can you please rename it to "Download Xray Results" just to be consistent with upload action?

Reply:
done